wpeverest/user-registration
# Fix - UI fixes after update of WP v5.3

### All Submissions:






### Changes proposed in this Pull Request:



This PR fixes overall design issues with the update of WP version 5.3

### How to test the changes in this Pull Request:

1. Update your WordPress to the version 5.3
2. Check the overall design on the dashboard.

### Types of changes:




### Other information:




### Changelog entry

> Fix - UI fixes after update of WP v5.3
